package.json

```diff
@@ -56,13 +64,17 @@
     "@fortawesome/free-brands-svg-icons": "^6.1.1",
     "@fortawesome/free-regular-svg-icons": "^6.1.1",
     "@fortawesome/free-solid-svg-icons": "^6.1.1",
+    "@ory/integrations": "^0.1.0",
+    "@ory/kratos-client": "^0.10.1",
+    "@oryd/hydra-client": "^1.9.0-alpha.3",
     "@serlo/authorization": "^0.42.0",
     "@serlo/legacy-editor-to-editor": "Entkenntnis/tmp-serlo-legacy-editor-to-editor",
     "@serlo/markdown": "Entkenntnis/tmp-serlo-markdown",
     "@tippyjs/react": "^4.0.0",
     "algebra.js": "^0.2.0",
     "array-move": "^4.0.0",
     "autoprefixer": "^10.0.0",
+    "axios": "^0.27.2",
```

Comment:
hint: don't use axios, use `fetch` directly in the frontend. next.js is polyfilling it automatically

docs: https://nextjs.org/blog/next-9-4#improved-built-in-fetch-support